todo_app/data/Card.py

```diff
@@ -1,23 +1,28 @@
 from datetime import datetime
-from todo_app.data.updateCard import UpdateCard
-import requests
+
+
 class Card:
-    def __init__(self,id, name, status, TrelloUrls, lastActivity = datetime.today()):
-        self.trello_urls = TrelloUrls
+    def __init__(self,id, name, status, mongoDB, lastActivity = datetime.today()):
+        self.mongoDB = mongoDB
```

Comment:
Might have been said in M2/3, but this pattern of having the Card (which primarily represents the object to be displayed) also contain logic for saving itself in Mongo feels like it's prone to confusion (and I ran into issues with this trying to suggest an approach for adding back the assertion to your integration test) - generally we want to keep that logic outside of the class itself where we can; even if it means this class becomes basically read only